content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+
+**Tetapkan** atribut tata letak menjadi `responsive` agar gambar diskalakan dan diubah ukurannya:
+
+```html
+<amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+```
+
+Bagus! Gambar kita memiliki aspek tinggi lebar yang benar dan secara responsif memenuhi lebar layar.
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-responsive.png', 412, 660, align='center third', caption="Gambar kini menjadi responsif!") }}
+
+{% call callout('Baca selengkapnya', type='read') %}
+Pelajari lebih lanjut Sistem Tata Letak AMP di [Spesifikasi Tata Letak AMP](/id/docs/design/amp-html-layout.html).
+{% endcall %}
+
+## Berhasil!
+
+Sekarang dokumen AMP Anda akan terlihat seperti berikut:
+
+```html
+<!doctype html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+    <link rel="canonical" href="/article.html">
+    <link rel="shortcut icon" href="amp_favicon.png">
+
+    <title>Artikel Berita</title>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        width: auto;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        background: Tomato;
+        color: white;
+        font-size: 2em;
+        text-align: center;
+      }
+
+      h1 {
+        margin: 0;
+        padding: 0.5em;
+        background: white;
+        box-shadow: 0px 3px 5px grey;
+      }
+
+      p {
+        padding: 0.5em;
+        margin: 0.5em;
+      }
+    </style>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <header>
+      Situs Berita
+    </header>
+    <article>
+      <h1>Nama Artikel</h1>
+
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam egestas tortor sapien, non tristique ligula accumsan eu.</p>
+
+      <amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+    </article>
+  </body>
+</html>
+```
+
+Muat ulang halaman dan lihat keluaran konsolnya. Anda seharusnya disambut dengan pesan berikut:
+
+<pre class="success-text">
+Validasi AMP berhasil.
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.